consensus/spos/interface.go

```diff
@@ -53,6 +53,8 @@ type ConsensusCoreHandler interface {
 	PrivateKey() crypto.PrivateKey
 	// SingleSigner returns the single signer stored in the ConsensusStore used for randomness and leader's signature generation
 	SingleSigner() crypto.SingleSigner
+	// KeyGenerator returns the key generator stored in the ConsensusStore
```

Comment:
ConsensusCore instead ConsensusStore in the whole file and also in some other files (2 files)

Reply:
done